test/soak/serve_hostnames/README.md

```diff
@@ -12,7 +12,7 @@ The pods are created individually (i.e. not with a replication controller).
 
 Here is some representative output.
 ```
-$ ./serve_hostnames
+./serve_hostnames
```

Comment:
I tended to prefer keeping command prompt because it is easy to distinguish executed command and the output.
However the guideline shows we need to separate command from output as https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output
So here should be applied this guideline for readable doc.

Reply:
@oomichi Thanks for your feedback. I've updated the code. Is that ok?